tests/node-token.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var vows = require('vows');
+var assert = require('assert');
+var fs = require('fs');
+var events = require('events');
+
+var CodeGen = require('../lib/codegen').CodeGen;
+
+vows.describe('Test Token').addBatch({
+    'Test Automatic Auth Token Assignment': {
+        topic: function(){
+            var swagger = JSON.parse(fs.readFileSync('tests/apis/token.json', 'UTF-8'));
```

Comment:
@borisirota can you rename this file to something more explicit like `protected.json`